src/client/Client.js

```diff
@@ -263,6 +263,8 @@ class Client extends BaseClient {
   /**
    * Logs out, terminates the connection to Discord, and destroys the client.
    * @returns {Promise}
+   * @example
```

Comment:
Does this _really_ need an example? :P

Reply:
I mean its a simple function but why not just add it in there

Reply:
it returns a Promise, so please atleast try to handle that promise with a simple console.log() or something

Reply:
@Dev-Yukine Good catch, will update sooner or later